ts/Stock/StockToolsGui.ts

```diff
@@ -1473,9 +1476,9 @@ class Toolbar {
         // Show hide toolbar
         this.showhideBtn = showhideBtn = createElement(DIV, {
             className: PREFIX + 'toggle-toolbar ' + PREFIX + 'arrow-left'
-        }, null as any, wrapper);
+        }, void 0, wrapper);
```

Comment:
Im pretty curios why we use `void 0` instead of an empty object `{}` ?

Reply:
When we don't have any styles to add, it is more effective to add undefined than to create an empty object, which internally in `createElement` will be truthy and passed over to `css`, where again it will be truthy and inspected for keys etc.